package/gluon-config-mode-domain-select/luasrc/lib/gluon/config-mode/wizard/0200-domain-select.lua

```diff
@@ -0,0 +1,48 @@
+return function(form, uci)
+	local site = require 'gluon.site'
+	local path = '/lib/gluon/domains'
+	local fs = require 'nixio.fs'
+	local json = require 'jsonc'
+
+	local function get_domain_list()
+		local list = {}
+		for domain_path in io.popen('find ' .. path .. '/ -iname \*.json -print'):lines() do
```

Comment:
- `\*` is no valid escape in Lua. Either use `\\*` to get a backslash in Shell, or replace all `'` with `"`, so you can just use `'*.json'` without any backslashes.
- use `-name` instead of `-iname`
- `-print` can be removed

... but even better would be to remove the whole *io.popen* and use *glob* from *nixio.fs* instead.